web/src/main/resources/public/src/app/admin/components/users/users.component.ts

```diff
@@ -44,18 +47,20 @@ export class UsersComponent implements OnInit {
     private houses: Array<string> = [];
     private apartment: Array<string> = [];
     genders = [
-        'male',
-        'female'
+        'Чоловік',
```

Comment:
Wrong. Should be an entity and retrieved from DB.
Internationalization also should work